ext/dapr-ext-fastapi/dapr/ext/fastapi/actor.py

```diff
@@ -26,6 +26,9 @@
 DEFAULT_CONTENT_TYPE = "application/json; utf-8"
 DAPR_REENTRANCY_ID_HEADER = 'Dapr-Reentrancy-Id'
 
+# This should be added to all magic Dapr Actor methods implemented here
+DEFAULT_ACTOR_ROUTER_TAGS: List[str] = ['Actor']
```

Comment:
Don't you think that putting this inside the actor class will be more convenient if someone will want to inherit the class and change the default tags?

Reply:
That's true. I changed the approach now.

As long as you call `super().__init__()` this should be fine now.